DigitalLearningSolutions.Web/Controllers/PricingController.cs

```diff
@@ -1,14 +1,17 @@
-﻿namespace DigitalLearningSolutions.Web.Controllers
-{
-    using DigitalLearningSolutions.Web.Attributes;
-    using Microsoft.AspNetCore.Mvc;
-    
-    public class PricingController : Controller
-    {
-        [RedirectDelegateOnlyToLearningPortal]
-        public IActionResult Index()
-        {
-            return View();
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class PricingController : Controller
+    {
+        [RedirectDelegateOnlyToLearningPortal]
+        [SetApplicationTypeAndSelectedTab(nameof(ApplicationType.Main), nameof(Tab.Pricing))]

```

Comment:
Is there any particular reason you've put this one on the action not the controller?